ui/src/pages/Edit/UploadStory.tsx

```diff
@@ -0,0 +1,721 @@
+import {
```

Comment:
@AbhiPrasad i used git mv but it still has massive diffs, im guessing I did it wrong? lmk what you think

Reply:
ya its fine whatever. Just rack up those contributions :)))

Reply:
yessir, i'll try again if it comes up.